src/content/docs/teams.mdx

```diff
@@ -1,46 +1,138 @@
+
+Organizations are a way to group apps and team members together. To create an
+organization, visit the Shorebird console and click the account dropdown in the
+top left corner. From there, click the "+" button next to "Organizations".
+
+![Create
+Organization](https://github.com/user-attachments/assets/6117e85f-ff07-4ffa-9683-3b4543f252d1)
+
+This will automatically create a new organization called "My Organization". You
+can head over to the organization's settings to change the name:
+
+![Rename
+Organization](https://github.com/user-attachments/assets/c33eda7c-bdd3-4ef2-ab0e-cbcaadbdd99f)
+
+## Managing Members
+
+To add one or more members to your organization, visit the organization's
+settings and click the "Add member" button.
+
+![Add
+Member](https://github.com/user-attachments/assets/584703fc-d20d-49b2-8358-b3b187c7fefc)
+
+You can also edit a member's role or remove them from the organization from the
+same organization settings page.
+
+![Organization Settings with new
+Member](https://github.com/user-attachments/assets/4936b211-23d8-4a44-90e6-06fab421a4f4)
+
+## Managing Organization Apps
+
+Organizations can have multiple apps associated with them. You can either
+transfer existing apps to an organization or create new apps within the
+organization directly.
+
+To transfer an existing app to an organization, visit the app's settings and
+click the "Transfer app" button. You can then select the organization you want
+to transfer the app to from the dropdown.
+
+![Transfer
+App](https://github.com/user-attachments/assets/9d7c5428-ebfd-4c9c-9777-4ca06da8d962)
+
+To create a new app within an organization, select the organization when running
+`shorebird init`:
+
+![Shorebird Init with
+Organization](https://github.com/user-attachments/assets/6a07fb1f-1fc6-4b41-9df8-60a50c42b475)
+
+## Organization Roles
+
+Organization roles apply to all apps within the organization and can be managed
+via the organization settings page in the Shorebird console.
+
+If you have any questions about access control for your app, please do not
+hesitate to reach out to contact@shorebird.dev.
+
+### Organization Member
+
+New members added to an organization are assigned the "Member" role by default.
+
+Members can:
+
+- Read information for all apps within the organization
+- View collaborators on all apps within the organization
+- Create releases and patches for all apps within the organization
+- View insights for all apps within the organization
+
+### Organization Admin
+
+Organization Admins can:
+
+- Everything a Member can do
+- Add and Remove Members for the organization
+- Delete Apps within the organization
+- Delete Releases within the organization
+
+### Organization Owner
+
+There is only one owner per organization (the person who created the
+organization) and they have full control over the organization and all apps
+within it.
+
+The owner can:
+
+- Everything an Admin can do
+- Manage the billing information for the organization
```

Comment:
There is an upsell opportunity to link to the pricing page and/or mention "If you need more control than this, consider our enterprise plan" or similar.